.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
 python:
   - "2.7"
+  - "3.4"
```

Comment:
We actually target Python 2.6 compatibility as well -- see this commit:
https://github.com/epantry/django-sql-explorer/commit/8b445b63fcd2d3dd1cb6bd36df119310c4112a25

Probably should have added that into Travis as well. Can you make sure 2.6 still runs and add it to travis?